src/appleseed.python/__init__.py

```diff
@@ -29,3 +29,7 @@
 
 from _appleseedpython import *
 from logtarget import *
+
+import logging
```

Comment:
We should not initialize logging here. Users importing appleseed.python might not need logging and if they need it, they probably want to initialize it themselves with a config that makes sense for them.

For now, lets move this code to each place where we use logging until we find a better way to package the texture converter tool.